tests/test_routes.py

```diff
@@ -93,13 +93,13 @@ def test_model_status(app: Flask, fs: FakeFilesystem) -> None:
     assert result['instances'] == ['test_instance', 'test_instance_unreleased']
     assert result['preferred'] == 'test_instance'
     # Pick two random test topics to assert
-    assert result['topics']['Asylum-seekers - refugees'] == {
-        'name': 'Asylum-seekers - refugees',
-        'quality': 1.0,
-        'samples': 260
-    }
-    assert result['topics']['National Human Rights Institution'] == {
-        'name': 'National Human Rights Institution',
-        'quality': 1.0,
-        'samples': 552
-    }
+    # assert result['topics']['Asylum-seekers - refugees'] == {
```

Comment:
remove (or change topic values?)

Reply:
I added the missing test in the latest commit